credentials/src/main/kotlin/model/SSITypes.kt

```diff
@@ -173,3 +175,25 @@ public class FilterV2(
   public val pattern: String? = null,
   public val type: String
 )
+
+/**
+ * Represents a presentation submission object.
+ *
+ * @see [Presentation Submission](https://identity.foundation/presentation-exchange/spec/v2.0.0/#presentation-submission)
+ */
+public data class PresentationSubmission(
+  val id: String,
+  val definitionId: String,
+  val descriptorMap: List<DescriptorMap>
+)
+
+/**
+ * Represents descriptor map for a presentation submission.
+ */
+@JsonInclude(JsonInclude.Include.NON_NULL)
```

Comment:
Curious what ths does?

Reply:
if fixed our json serialization so that if something is null it just does not show up in a jsonObject:
```
vp :{
  vc : {...},
  format: null
}

VS

vp :{
  vc : {...}
}
```